doc/howto/first_ab_experiment.md

```diff
@@ -0,0 +1,62 @@
+# How to run the initial AB Experiment in India Production 
+
+### Date: July 2021
+### References
+* [clubhouse story ch3586](https://app.clubhouse.io/simpledotorg/story/3586/test-in-sandbox)
+* [GitHub PR](https://github.com/simpledotorg/simple-server/pull/2752)
+
+---
+
+
+*Note: these instructions are only valid for Tuesday July 27 before 4pm IST (the end of the messaging window).*
+
+1. Turn on `experiment` and `whatsapp_appointment_reminders` Flipper feature flags
+2. Merge https://github.com/simpledotorg/simple-server/pull/2752 - it includes a migration that will create two new experiments with three treatment groups each. To verify:
+
+```ruby
+experiments = Experiment.where(name: "Small Current Patient July 2021")
+experiments.count # should return 2
+experiments.map { |e| e.treatment_groups.count } # should return 3
+```
+
+3. Create the current patient experiment, based on how many patients are eligible. We want to use approximately 300 patients and this particular job uses percentages. So if 10,000 patients are eligible, we would want to use 3% (300/10,000). 
+**NOTE:** change the percentage below so that the experiment enrolls ~300 patients
+
+  ```ruby
+  Time.zone = Period::REPORTING_TIME_ZONE
+
+  candidates = ExperimentControlService.send(:current_patient_candidates, "July 28, 2021".to_date, "July 30, 2021".to_date).count
+  # => 10000 ...assume there are 10,000 eligible patients returned...
+  percentage = 300 / candidates # results in 0.03
+  ExperimentControlService.start_current_patient_experiment("Small Current Patient July 2021", 1, 3, percentage) # the final argument is the percentage of patients to enroll
+
+  # Verification
+  experiment = Experiment.find_by!(name: "Small Current Patient July 2021")
+  experiment.notifications.count
+  # => should return around 400 notifications
```

Comment:
by commenting out notes like this, it means the operator can copy paste blocks of commands and it won't execute the docs